components/centraldashboard/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions":{
         "module":"commonjs",
         "esModuleInterop":true,
-        "target":"es6",
+        "target":"es2020",
         "noImplicitAny":true,
```

Comment:
Change to `es2020` because of the latest K8s Node.js client used `BigInt` (Node.js 12 supported).